app/models/viewmodels/SubmissionTabItem.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.viewmodels
+
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.tabs.{TabItem, TabPanel}
+
+case class SubmissionTabItem(tab: String) {
```

Comment:
It seems a bit over-engineered a case class here. Wouldn't be enough just to have in the Dashboard template something like ...
```
@tabItem(tab: String, content: HtmlContent) = @{
  TabItem(id = Some(s"$tab-submissions"), label = messages(s"submissions.$tab.tab.title"), panel = TabPanel(content = content))
}
```